lib/turbolinks.rb

```diff
@@ -7,19 +7,27 @@
 require 'turbolinks/redirection'
 
 module Turbolinks
+  module ControllerConcern
```

Comment:
> IMO it's confusing since that module isn't a controller. It's "something you add to your controller".

I think that's just the nature of ruby module naming. Technically, nor is `Enumerable` an enumerable... it's equally "a set of features you add to something that *is already* enumerable (i.e., implements `each`)".